src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -92,7 +92,7 @@ object JesBackend {
    * @return A path which is unique per input path
    */
   def localFilePathFromCloudStoragePath(gcsPath: GcsPath): Path = {
-    Paths.get(JesWorkingDisk.MountPoint).resolve(gcsPath.bucket).resolve(gcsPath.objectName)
+    Paths.get(gcsPath.bucket).resolve(gcsPath.objectName)
```

Comment:
JES works with relative paths now (relative to the disk they're localized to), not absolute paths